build_scripts/start_course.sh

```diff
@@ -9,20 +9,30 @@ else
   exit 1
 fi
 
+# If the DOWNLOAD variable isn't set, default it to 0
```

Comment:
It says 1 in the if block

Reply:
Good catch, the code here is correct and the comment is not.  At least I think it makes sense that if someone new were working on this, they might not have an external copy of `ocw-to-hugo` so download using the dependency version should be the default.